src/components/Home.js

```diff
@@ -1,10 +1,39 @@
 import React from "react";
+import { NavLink } from "react-router-dom";
 import "./Home.css"
+import img1 from "../images/wineGlasses.jpeg"
+import featImg1 from "../images/goatCheese.jpg"
+import featImg2 from "../images/wineBottle.jpg"
+import featImg3 from "../images/Oscypki.jpg"
+import mainLogo from "../images/mainLogo.png"
 
 const Home = () => {
     return <>
-      <h1>Home</h1>
+      <div className="componentContainerHome">
+        <div className="featuredContainer">
+            <img src={img1} className="img1"/>
+        </div>
+        <div className="featured">
+            <div>
+                <h1>Featured Products</h1>
+            </div>
+            <div className="fpCards">
+                <div className="fp">
```

Comment:
If we could write out the abbreviations, that may help others trying to read the code.